Documentation/kubernetes-on-aws.md

```diff
@@ -178,6 +194,8 @@ If deploying a production Kubernetes cluster, consider establishing PKI independ
 
 #### Configure your DNS
 
+This step is not necessary if `createRecordSet` is true. In this case, a host record has already been created for you.
```

Comment:
Idea: instead of referencing the feature by its config name, we use Route 53. Like:

> If you configured Route 53 settings in your configuration above, a host record has already been created for you.

Probably need an "otherwise" in the line below:

> **Otherwise**, navigate to the...